addons/auth/tests/helpers/index.js

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
Do we need this empty test?

Reply:
This is a test helper.  It comes with new Ember 4 addons so I left it.

Reply:
Dang I missed and confused for a test 😅